sdk/include/sdk/Config.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2023 Robert Bosch GmbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef VEHICLE_APP_SDK_CONFIG_H
+#define VEHICLE_APP_SDK_CONFIG_H
+
+#include "middleware/Middleware.h"
+
+namespace velocitas {
+
+class Config {
```

Comment:
It looks like factory or provider, but not as a config :)

Reply:
Yeah, not sure about this. Maybe we should remove it again. It is also there in Python, but I must admit that I didn't spent so much thinking about this.